wagtail/admin/templates/wagtailadmin/pages/side_panels/includes/side_panel_button.html

```diff
@@ -8,9 +8,10 @@
     has_toggle - Show toggle icon on the button
     attr - custom attributes for the button
 {% endcomment %}
-<button type="button" class="{{ classes }} w-bg-transparent w-text-14 w-p-0 w-text-teal-200 hover:w-text-teal-600 w-inline-flex w-justify-center w-transition" {% if data_url %}data-url="{{ data_url }}" {% endif %}{% if has_toggle %}data-button-with-dropdown-toggle {% endif %}{{ attr }}>
-    {{ text }}
-    {% if has_toggle %}
-        {% icon name='arrow-down' class_name='w-w-4 w-h-4 w-transition motion-reduce:w-transition-none' %}
-    {% endif %}
-</button>
+{% spaceless %}
+    <button type="button" class="{{ classes }} w-bg-transparent w-text-14 w-p-0 w-text-teal-200 hover:w-text-teal-600 w-inline-flex w-justify-center w-transition empty:w-hidden" {% if data_url %}data-url="{{ data_url }}" {% endif %}{% if has_toggle %}data-button-with-dropdown-toggle {% endif %}{{ attr }}>{{ text }}
```

Comment:
can the `{{ text }}` be on a new line - makes it easier to read what is being done here, especially with the super long classes.

```suggestion
    <button type="button" class="{{ classes }} w-bg-transparent w-text-14 w-p-0 w-text-teal-200 hover:w-text-teal-600 w-inline-flex w-justify-center w-transition empty:w-hidden" {% if data_url %}data-url="{{ data_url }}" {% endif %}{% if has_toggle %}data-button-with-dropdown-toggle {% endif %}{{ attr }}>
        {{ text }}
```